jr-objects/src/test/groovy/com/fasterxml/jackson/jr/passing/GroovyTest.groovy

```diff
@@ -0,0 +1,68 @@
+package com.fasterxml.jackson.jr.passing
```

Comment:
Is the new `passing` needed to avoid split packages or... ?

If necessary, maybe use `com.fasterxml.jackson.jr.ob.groovy`.

Reply:
my mistake, removed in latest commits.